docs/06_how-to-guides/10_compile/01_compile-a-contract-via-cli.md

```diff
@@ -1,20 +1,35 @@
 ---
-content_title: How to compile a contract via CLI
+content_title: How to compile a smart contract via CLI
 ---
 
-## Preconditions
-- You have the source of your contract saved in one of your local folders, e.g. `./examples/hello`
-For details on how to create your first contract follow [this tutorial here](https://developers.eos.io/eosio-home/docs/your-first-contract)
-
-Follow these steps to compile your contract:
-
-1. Navigate to the hello folder in examples (./examples/hello), you should then see the ./src/hello.cpp file
-2. Now run following commands:
-```sh
-$ mkdir build
-$ cd build
-$ eosio-cpp -abigen ../src/hello.cpp -o hello.wasm -I ../include/
-```
-3. This will generate two files:
-- The compiled binary wasm, hello.wasm
-- The generated ABI file, hello.abi
+## Overview
+
+This guide provides instructions how to compile a smart contract using the command line interface (CLI).
+
+## Before you begin
+
+* You have the source of the contract saved in a local folder, e.g. `./examples/hello/`
+For details on how to create your first contract follow the [Hello World Contract](https://developers.eos.io/welcome/latest/smart-contract-guides/hello-world) guide.
+
+## Procedure
+
+Follow the following steps to compile your contract.
+
+1. Navigate to the hello folder in examples `./examples/hello`. You should see the `./src/hello.cpp` file.
+
+2. Run the following commands:
+
+    ```sh
+    mkdir build
+    cd build
+    eosio-cpp -abigen ../src/hello.cpp -o hello.wasm -I ../include/
```

Comment:
Does this need the where section too?

Reply:
i.e. 
where

-abigen = 
../src/hello.cpp = 
etc

Reply:
yes and a reference section... I am slowly getting the feeling this is a never ending task